package/MDAnalysis/coordinates/RDKit.py

```diff
@@ -59,7 +59,25 @@
 import numpy as np
 
 from . import memory
-
+from . import base
+
+try:
+    from rdkit import Chem
+except ImportError:
+    pass
+else:
+    RDBONDTYPE = {
+        'AROMATIC': Chem.BondType.AROMATIC,
+        'SINGLE': Chem.BondType.SINGLE,
+        'DOUBLE': Chem.BondType.DOUBLE,
```

Comment:
Duplicate of #

Reply:
Any chance you could elaborate here @good-luck123? It's not immediately clear to me as to what you mean.